examples/connectivity/plot_adhd_spheres.py

```diff
@@ -0,0 +1,72 @@
+"""
+Multi-subject connectome for ADHD
+=================================
+
+This example estimates a connectome from spheres in the brain. The coordinates
+of the spheres have been taken from a meta-analysis on neurosynth website.
+
+"""
+import matplotlib.pyplot as plt
+
+from nilearn import plotting
+
+
+n_subjects = 4  # subjects to consider for group-sparse covariance (max: 40)
```

Comment:
Sorry: my remark is incomprehensible.

I think that, to make the example simple, we should not use a group sparse covariance, by something simpler: GraphLasso maybe.